tests/examples/test_examples.py

```diff
@@ -110,3 +114,42 @@ def test_example_sample_lib():
 
             # Client worked well, let's stop the server
             server.terminate()
+
+
+def test_lazy_load_local_models_invalid_model_dir():
+    """Make sure a ValueError is raised,
+    when lazy_load_local_models is set to True but local_models_dir does not exist.
+    """
+
+    invalid_local_models_dir = path.abspath(
+        path.join(path.dirname(__file__), "invalid")
+    )
+
+    with TemporaryDirectory() as workdir:
+
+        caikit.config.configure(
+            config_dict={
+                "merge_strategy": "merge",
+                "runtime": {
+                    "library": "sample_lib",
+                    "local_models_dir": invalid_local_models_dir,
+                    "lazy_load_local_models": True,
```

Comment:
can you add 4 tests with `local_models_dir` false and true, `lazy_load_local_models` false and true.